lib/flate.js

```diff
@@ -6,8 +6,10 @@ exports.uncompressInputType = USE_TYPEDARRAY ? "uint8array" : "array";
 exports.compressInputType = USE_TYPEDARRAY ? "uint8array" : "array";
 
 exports.magic = "\x08\x00";
-exports.compress = function(input) {
-    return pako.deflateRaw(input);
+exports.compress = function(input, compressionOptions) {
+    return pako.deflateRaw(input, {
+        level : compressionOptions.level || -1 // default compression
```

Comment:
The `generate` method use an empty object as a fallback : https://github.com/dduponchel/jszip/blob/889d1124362b3b5df5a7e1c7321635df7af7a3e0/lib/object.js#L676